features/upgrade/web/console-upgrade.feature

```diff
@@ -88,3 +88,25 @@ Feature: web console related upgrade check
     When I perform the :check_resource_item_name web action with:
       | resource_name | hello-daemonset |
     Then the step should succeed
+
+  # @author yapei@redhat.com
+  # @case_id OCP-76051
+  @upgrade-check
+  @admin
+  @console
+  @users=upuser1,upuser2
+  @4.16 @4.15 @4.14 @4.13 @4.12 @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
```

Comment:
The rules are added for 4.14 to 4.18, while tags here does not have 4.17 and 4.18.  Is it expected ?

Reply:
good catch, let me update